pkg/volume/csi/csi_attacher.go

```diff
@@ -228,13 +228,19 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 		return errors.New(log("attacher.MountDevice failed, deviceMountPath is empty"))
 	}
 
+	corruptedDir := false
 	mounted, err := isDirMounted(c.plugin, deviceMountPath)
```

Comment:
I think actually we may want to skip this check altogether and just always call StagePublish. @gnufied @jsafrane do you see any problems with that? Ref https://github.com/kubernetes/kubernetes/issues/86784

Reply:
yes, we don't even need to do `os.MkdirAll(deviceMountPath, 0750)`, all leave csi driver to handle that logic:
https://github.com/kubernetes/kubernetes/blob/e4a5012de6b49ca7c3b10c4647205fdca66014ee/pkg/volume/csi/csi_attacher.go#L290-L293
this looks like a big behavior change, is it ok to do this in this PR?

Reply:
I would like to back port this fix to old release, so shall we go with this PR now?
about `removing the mounted check`, I could work out another PR that won't be back ported, is that ok?  @gnufid thanks.